### PR TITLE
fix: [Tree] Scrolling items jitters when using both virtualize and re…

### PR DESCRIPTION
**Pull request by boomboomchen**
…nderFullLabel #527

[中文模板 / Chinese Template](https://github.com/DouyinFE/semi-design/blob/main/.github/PULL_REQUEST_TEMPLATE.zh-CN.md)

- [ ] I have read and followed [Pull Request Guidelines](https://github.com/DouyinFE/semi-design/blob/main/CONTRIBUTING-en-US.md#pull-request-guidelines) of the contributing guide.

### What kind of change does this PR introduce? (check at least one)

 - [x] Bugfix
 - [ ] Feature
 - [ ] Code style update
 - [ ] Refactor
 - [ ] Test Case
 - [ ] TypeScript definition update
 - [ ] Document improve
 - [ ] CI/CD improve
 - [ ] Branch sync
 - [ ] Other, please describe:

### PR description

Fixes #

### Changelog
🇨🇳 Chinese
- Fix: 修复 Tree 同时使用虚拟化和 renderFullLabel 时，滚动项目发生抖动的问题 #527 

---

🇺🇸 English
- Fix: Fix the jitter of scrolling items when Tree uses both virtualization and renderFullLabel #527

### Checklist
- [ ] Test or no need
- [ ] Document or no need
- [ ] Changelog or no need

### Other
- [ ] Skip Changelog

### CODE REVIEW

File: packages/semi-ui/tree/treeNode.tsx [hunk 1]
```diff
@@ -392,7 +392,14 @@ export default class TreeNode extends PureComponent<TreeNodeProps, TreeNodeState
                     ...dragProps
                 });
             } else {
-                return customLabel;
+                if (isEmpty(style)) {
```
Comment (shijiatongxue) [inline packages/semi-ui/tree/treeNode.tsx:395]: 这里之前应该是需要用户手动传的。

image

Comment (shijiatongxue) [reply packages/semi-ui/tree/treeNode.tsx:395]: 现在这种做法是不需要用户手动传了是吗？

Comment (boomboomchen) [reply packages/semi-ui/tree/treeNode.tsx:395]: 是的

Comment (shijiatongxue) [reply packages/semi-ui/tree/treeNode.tsx:395]: ok

**Pull request merged**